templates/fine-tune-llm_v2/end-to-end-examples/fine-tune-preference/README.ipynb

```diff
@@ -0,0 +1,1028 @@
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "mode: eval\n",
+      "input_folder: s3://air-example-data/preference-tuning-summarization-example/qa_generation/qa_annotations_full_test\n",
+      "inference_type: offline\n",
+      "model_inference_config:\n",
+      "  # Modify with s3 link to full param weights if you did full-param training\n",
+      "  model_id_or_path: mistralai/Mistral-7B-Instruct-v0.1\n",
+      "\n",
+      "  # Add path to lora weights here. If you did full param training, you can instead remove this field.\n",
+      "  adapter_id_or_path: s3://large-dl-models-mirror/finetuning_template/mistral_dpo_summarization_lora\n",
+      "\n",
+      "  temperature: 0\n",
+      "  top_p: 0.95\n",
+      "  scaling_config:\n",
+      "    batch_size: 64\n",
+      "    concurrency: 2\n",
+      "    num_gpus_per_instance: 2\n",
+      "    accelerator_type: A10G\n",
+      "num_generations: 1\n",
+      "judge_inference_config:\n",
+      "  model_id_or_path: meta-llama/Meta-Llama-3.1-8B-Instruct\n",
+      "  temperature: 0\n",
+      "  scaling_config:\n",
+      "    batch_size: 64\n",
+      "    concurrency: 3\n",
+      "    num_gpus_per_instance: 2\n",
+      "    accelerator_type: A10G\n",
+      "num_mcq_questions: 5\n"
+     ]
+    }
+   ],
+   "source": [
+    "!cat configs/summary_generation/8b_judge/mistral_finetuned_eval.yaml"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    " 🔄 REPLACE the `adapter_id_or_path` entry in the config with the path to your LoRA weights before proceeding (if you used the fine-tuning defaults). Alternatively, make sure to replace `model_id_or_path` entry (and remove the `adapter_id_or_path` entry) if you did full-param fine-tuning."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We are now ready to evaluate our fine-tuned model: \n",
+    "\n",
+    "```bash\n",
+    "anyscale job submit -f configs/jobs/8b_judge/generate_summaries_eval_finetuned_job.yaml\n",
+    "# Optional: use the 70b model for better performance (runs on A100s)\n",
+    "# anyscale job submit -f configs/jobs/70b_judge/generate_summaries_eval_finetuned_job.yaml\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In the logs for the above jobs, you should see the final path to the output summaries for both the models. \n",
+    "\n",
+    "Optionally, you can also obtain the summaries and scores for the `gpt-4o` model from OpenAI. Simply run: \n",
+    "\n",
+    "```bash\n",
+    "anyscale job submit -f configs/jobs/8b_judge/generate_summaries_eval_gpt_job.yaml\n",
+    "# Optional: use the 70b model for better performance (runs on A100s)\n",
+    "# anyscale job submit -f configs/jobs/70b_judge/generate_summaries_eval_gpt_job.yaml\n",
+    "```\n",
+    "\n",
+    "This should take about ~10 min for the 8B model and the 70B model"
```

Comment:
How many resources will it required for each? again user needs to be able to compute the cost.

Reply:
Added!